contracts/liquidity_hub/pool-manager/src/liquidity/commands.rs

```diff
@@ -98,19 +120,19 @@ pub fn provide_liquidity(
 
                 share
             } else {
```

Comment:
if there's no else case can we remove it?

Reply:
There's an else case